python/ray/_raylet.pyx

```diff
@@ -1421,6 +1426,32 @@ cdef class CoreWorker:
         logger.warning("Local object store memory usage:\n{}\n".format(
             message.decode("utf-8")))
 
+    cdef python_scheduling_strategy_to_c(
+            self, python_scheduling_strategy,
+            CSchedulingStrategy *c_scheduling_strategy):
+        cdef:
+            CPlacementGroupSchedulingStrategy \
+                *c_placement_group_scheduling_strategy
+        if python_scheduling_strategy is None:
+            c_scheduling_strategy[0].clear_scheduling_strategy()
```

Comment:
Is `clear_scheduling_strategy ` a protobuf method, or setting the nullptr for the `c_scheduling_strategy `?

I think it is more like at the end of this method, is `c_scheduling_strategy ` guaranteed to be not null? (can you add assert?)

Reply:
yes, it's a protobuf method: https://developers.google.com/protocol-buffers/docs/reference/cpp-generated#oneof

c_scheduling_strategy can be "null" (ONEOF_NOT_SET) indicating we want to use the default scheduling strategy.